### ISSUE TITLE
NetNewsWire is not able to sync read articles and subscriptions

### ISSUE THREAD

**Issue by raveensrk** [Ranchero-Software/NetNewsWire#3592]
I also dont see the NetNewsWire Directory in the icloud drive in my macos. I think it used to be there. When I look at the accounts in preferences the icloud account is active on both macos and ios.

Not sure what other information that I should provide.

**Comment by vincode-io** [comment]
What do you mean by "NetNewsWire Directory"? We don't use iCloud Drive for anything in NetNewsWire, so there shouldn't be anything there anyway.

**Comment by raveensrk** [comment]
Oh ok. I thought it used one like some other apps. Anyway, What should I do to debug this problem? Should I reinstall the apps? What information should I provide?

**Comment by raveensrk** [comment]
@vincode-io I went ahead and deleted my macos newswire icloud account and added it again. Now its able to sync properly. I am not sure what caused the issue in the first place. It could be because my macos ran out of storage space many times this last 2 weeks and I had to delete some large media files to get some space. Maybe NetNewsWire was not able to download the sync data due to storage space issues.

Let me know what you think.
